src/plotman/manager.py

```diff
@@ -128,6 +128,9 @@ def maybe_start_new_plot(dir_cfg, sched_cfg, plotting_cfg):
             if dir_cfg.tmp2 is not None:
                 plot_args.append('-2')
                 plot_args.append(dir_cfg.tmp2)
+            elif plotting_cfg.use_dst_as_tmp2:
+                plot_args.append('-2')
+                plot_args.append(dir_cfg.dst)
 
```

Comment:
When tmp2 is same as dst dir, chia plotter won't copy the final file, it will just rename it.
Bases on this it would be better to use the value of 'dstdir' variable because if you have multiple dst dirs, these way you make sure the tmp2 is the same as dstdir for this specific plot, and thus avoid an additional copy at the end.